src/js/common/components/CampaignSupport/PayToPromoteProcess.jsx

```diff
@@ -354,58 +345,58 @@ class PayToPromoteProcess extends Component {
             </ContributeGridWrapper>
           </InnerWrapper>
         </OuterWrapper>
-        <PaymentWrapper>
```

Comment:
Hi @SailingSteve we actually are using this code for a UX test and can't strip this out at this time. I'd rather leave PayToPromoteProcess untouched this close to the election. I'm happy to jump on a screenshare if you have any questions.